Libraries/pbxbuild/Headers/pbxbuild/Target/Environment.h

```diff
@@ -50,15 +50,30 @@ class Environment {
     std::vector<std::string>                 _architectures;
```

Comment:
I know little about pbxsettings, so a lot of this is over my head.  That said, tests are good, and no obvious bugs stood out to me, so LGTM.